src/wallet/wallet_node.py

```diff
@@ -349,9 +349,10 @@ def has_full_node(self) -> bool:
 
     async def new_peak(self, peak: wallet_protocol.NewPeak, peer: WSChiaConnection):
         assert self.wallet_state_manager is not None
-        curr_peak = self.wallet_state_manager.blockchain.get_peak()
```

Comment:
Why are we removing this code?

Reply:
`curr_peak.weight > curr_peak.weight` That's impossible to be true. I think you mean to compare it to something else but not completely clear from context what that something else is.

Reply:
and if you don't test curr_peak then it's an assignment without use.

Reply:
oh yeah, it should be peak.weight > curr_peak.weight @Yostra